File: Haskell/Questionários/Extras/A6.py
def divida(l):
    
    r = 0
    i = 0
    
    while i < len(l):
        emp = l[i][0]
        juros = l[i][1]
        
        r += emp 
        r *= (1 + juros/100)
        
        i += 1
        
    return r

print(f'{divida([(10,10), (9,20), (-10,0)])}')

File: Haskell/Questionários/Extras/A6.hs
divida :: [(Int,Int)] -> Int
divida l = foldl f 0 l where
        f acc r = (fst r + acc) * (1 + snd r / 100)

emprestimos :: [(Int, Int)]
emprestimos = [(10,10), (9,20), (-10,0)]

main :: IO ()
main = print (divida emprestimos)